Harvest.Core/Models/AccessCodes.cs

```diff
@@ -11,13 +11,18 @@ public static class AccessCodes
         public const string FinanceAccess = "FinanceAccess";
         public const string ReportAccess = "ReportAccess";
         public const string InvoiceAccess = "InvoiceAccess"; //Basically the same as PI, but need to add FinanceRole to just the invoice page.
+        public const string ProjectAccess = "ProjectAccess"; //Need to allow finance to see projects in addition to the workers, but not enter expenses.
         /// <summary>
         /// When using this Authentication, the projectId must be in the parameters.
         /// </summary>
         public const string PrincipalInvestigator = "PrincipalInvestigator";
         /// <summary>
         /// When using this Authentication, the projectId must be in the parameters.
         /// </summary>
-        public const string PrincipalInvestigatorOnly = "PrincipalInvestigatorOnly ";
+        public const string PrincipalInvestigatorOnly = "PrincipalInvestigatorOnly ";

```

Comment:
I know it was here before, but this has a trailing space and we should get rid of it, i think it'll only cause confusion.